### PR TITLE
add huggingface exporter

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##
add hugging face import option. Refactor the MetaBuilder architecture

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

File: model-uploader/python/pytorch/huggingface_import.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    export_path = os.path.join(export_path, name)
+    if not os.path.exists(export_path):
+        os.makedirs(export_path)
+    traced_model.save(os.path.join(export_path, name + '.pt'))
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+    tensors = []
+    for shape in eval(args.shape):
+        tensors.append(torch.ones(shape))
```
Comment (stu1130) [inline model-uploader/python/pytorch/huggingface_import.py:35]: I am not sure if ones represents the real input
at least for bertqa, input_id would not be the ones
```
encoded_dict = tokenizer.encode_plus(sequence_a, sequence_b)

assert encoded_dict['input_ids'] == [101, 20164, 10932, 2271, 7954, 1110, 1359, 1107, 17520, 102, 2777, 1110, 20164, 10932, 2271, 7954, 1359, 136, 102]
assert encoded_dict['token_type_ids'] == [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1]
```

Comment (stu1130) [reply model-uploader/python/pytorch/huggingface_import.py:35]: Also different kind of NLP model may required different input, some need attention_mask, some don't

Comment (lanking520) [reply model-uploader/python/pytorch/huggingface_import.py:35]: This is just the fake input passed in the model.

Comment (stu1130) [inline model-uploader/python/pytorch/huggingface_import.py:35]:
```suggestion
        tensors.append(torch.ones(shape, dtype=torch.int64))
```
I think at least we should match the dtype
all the the inputs for BERT are int64

**Pull request merged**